### DISCUSSION TITLE
Can we open a file in the last cursor positions?

### DISCUSSION THREAD

**Discussion by joaocarvalhoopen** [helix-editor/helix#4605] [General] [answered]
Hello,

In Helix is the a way to make Helix remember the last editing position or last cursor position inside a file that is open again?

I searched the documentation but could find anything about it.

Thank you,

Best regards,
João

**Comment by Ordoviz** [comment, marked as answer]
Not yet. This feature requires #401

**Comment by joaocarvalhoopen** [comment]
Thank you.
